CMakeLists.txt

```diff
@@ -105,8 +110,13 @@ add_feature_info(WITH_SANITIZERS WITH_SANITIZERS "Build with address sanitizer a
 add_feature_info(WITH_MSAN WITH_MSAN "Build with memory sanitizer")
 add_feature_info(WITH_FUZZERS WITH_FUZZERS "Build test/fuzz")
 add_feature_info(WITH_NEW_STRATEGIES WITH_NEW_STRATEGIES "Use new strategies")
-if(BASEARCH_ARM_FOUND)
-    add_feature_info(WITH_ACLE WITH_ACLE "Build with ACLE CRC")
+if(BASEARCH_X86_FOUND)
+    add_feature_info(WITH_AVX2 WITH_AVX2 "Build with AVX2")
+    add_feature_info(WITH_SSE2 WITH_SSE2 "Build with SSE2")
+    add_feature_info(WITH_SSE4 WITH_SSE4 "Build with SSE4")
+    add_feature_info(WITH_PCLMULQDQ WITH_PCLMULQDQ "Build with PCLMULQDQ")
+elseif(BASEARCH_ARM_FOUND)
```

Comment:
I see no reason why BASEARCH_ARM_FOUND shouldn't be the first as A is first letter in Latin alphabet.
Changing order of existing tests clutters the patch unnecessarily.

Reply:
Good catch. I have corrected it.